https://issues.redhat.com/browse/ACM-10611 (#6292)
open-cluster-management/rhacm-docs: troubleshooting/trouble_compliance_history.adoc

[#troubleshooting-gov-framework]
= Troubleshooting compliance events in the compliance history API (Technology Preview)

> 

Reviewer: I think we don't need the tech preview tag on a troubleshooting topic. The reason why: That is used in the title and nav to highlight new function. I think since we are highlighting here a work around, we may not want that tag....to me it suggests that the actual workaround is tech preview. 

You can always put a line in the content in a spot like this:

...which indicates the policy compliance history API (Technology Preview) could not be reached

> If policy compliance events are not being recorded for a particular managed cluster, you may look at the `governance-policy-framework` logs on the affected managed cluster:

Author: ```suggestion
Some policy compliance events are not being recorded for a particular managed cluster.
```

> 
> ----
> oc -n open-cluster-management-agent-addon logs deployment/governance-policy-framework -f
> ----

Author: ```suggestion
```

> You might see a message value that is empty in the log message, which indicates the policy compliance history API could not be reached. A likely cause is the policy compliance history API URL is not correct or there is a network communication issue.

Reviewer: A possible cause?
Not sure we use "likely" given we don't use the word "like" much.

----
024-03-05T19:28:38.063Z        info    policy-status-sync      statussync/policy_status_sync.go:750    Failed to record the compliance event with the compliance API. Will requeue.       {"statusCode": 503, "message": ""}
----

[#resolving-gov-framework]
== Resolving the problem: Check the governance policy framework log

> Look at the `governance-policy-framework` logs on the affected managed cluster with the following command:

Author: ```suggestion
View the `governance-policy-framework` logs on the affected managed cluster with the following command:
```

> 

Reviewer: View the ....logs

== Resolving the problem: Check the governance policy framework log

> //still workin on this

Author: ```suggestion
Look at the `governance-policy-framework` logs on the affected managed cluster with the following command:

----
oc -n open-cluster-management-agent-addon logs deployment/governance-policy-framework -f
----
```
